src/app/api/imageInput/route.ts

```diff
@@ -0,0 +1,49 @@
+import { ChatOpenAI } from "@langchain/openai";
+import { HumanMessage } from "@langchain/core/messages";
+import { env } from "@/app/env.mjs";
+import { NextResponse } from "next/server";
+
+const chat = new ChatOpenAI({
+  modelName: "gpt-4-vision-preview",
+  maxTokens: 1024,
+  openAIApiKey: env.OPEN_AI_API_KEY,
+});
+
+export async function POST(request: Request) {
+  let formData = await request.formData();
```

Comment:
send whole messages array to the api for full chat context

Reply:
@Rohittomar01

Reply:
have sent